litgpt/model.py

```diff
@@ -244,6 +260,25 @@ def forward(
                 raise TypeError("You need to call `gpt.set_kv_cache()`")
             k, v = self.kv_cache(input_pos, k, v)
 
+        # In Gemma only layers 0, 2, 4, ... have a sliding window attention
+        if self.config.sliding_window_size is not None and not self.block_idx % 2:
```

Comment:
Big kudos for implementing this. We could also apply this to Mistral now (separate PR)

Reply:
As I stated about, I forgot to make changes so other models with SWA on every layer (like Mistral) can use it.
Right now it's tied to Gemma v2.